tic-tac-toe-app/App.jsx

```diff
@@ -1,5 +1,87 @@
+function Cells(props) {
+  return <button onClick={props.click} className='button'>{props.value}</button>
+}
+
+function Board(props) {
+  const [boardState, setBoardState] = React.useState(new Array(9).fill(null));
+
+  function declareWinner(board) {
+    const conditions = [
+      [0, 1, 2],
+      [3, 4, 5],
+      [6, 7, 8],
+      [0, 3, 6],
+      [1, 4, 7],
+      [2, 5, 8],
+      [0, 4, 8],
+      [2, 4, 6],
+    ];
```

Comment:
Proactively setting up winning conditions allows helps to communicate the the programmer's decision very clearer or other developers. 👍